README.md

```diff
@@ -127,14 +127,25 @@ http://localhost:3000/en/map/eleprofile/qdjango/2 # g3w-client (development)
 If you want develop custom plugins you need to place them in the [`src/plugins`](https://github.com/g3w-suite/g3w-client/blob/dev/src/plugins) folder, below you can see some examples:
 
 - [base-template](https://github.com/g3w-suite/g3w-client-plugin-base-template)
-- [editing](https://github.com/g3w-suite/g3w-client-plugin-editing)
 - [eleprofile](https://github.com/g3w-suite/g3w-client-plugin-elevation-profile)
 - [openrouteservice](https://github.com/g3w-suite/g3w-client-plugin-openrouteservice)
-- [qplotly](https://github.com/g3w-suite/g3w-client-plugin-qplotly)
-- [qtimeseries](https://github.com/g3w-suite/g3w-client-plugin-qtimeseries)
 - [queryresult-template](https://github.com/g3w-suite/g3w-client-plugin-queryresult-template)
 - [sidebar-template](https://github.com/g3w-suite/g3w-client-plugin-sidebar-template)
 
+### Default Plugins
+
+- [editing](https://github.com/g3w-suite/g3w-client-plugin-editing)
+- [qplotly](https://github.com/g3w-suite/g3w-client-plugin-qplotly)
+- [qtimeseries](https://github.com/g3w-suite/g3w-client-plugin-qtimeseries)
+
+To install Default G3W-SUITE plugin run follow command:
+
+```sh
+
+git submodules init      # to initialize your local configuration file
+git submodule update     # to fetch all the data from that project and check out the appropriate commit listed in your project
```

Comment:
@volterra79 on which `git` version did you test the above commands?

Reply:
@Raruto git version 2.17.1

Reply:
I've read about different "default" behaviors depending on the installed version, actually mine are:

```log
git version 2.39.0.windows.2 # windows 10
git version 2.34.1           # windows subsystem for linux (ubuntu jammy)
```

Just to know, have you already updated your git version to the latest available on your OS?